x-pack/plugins/observability/public/hooks/use_messages_storage.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback } from 'react';
+import { useKibana } from '../utils/kibana_react';
+
+export interface UseMessagesStorage {
+  getMessages: (plugin: string) => string[];
+  addMessage: (plugin: string, id: string) => void;
+  removeMessage: (plugin: string, id: string) => void;
+  clearAllMessages: (plugin: string) => void;
+  hasMessage: (plugin: string, id: string) => boolean;
+}
+
+export const useMessagesStorage = (): UseMessagesStorage => {
+  // @ts-ignore
```

Comment:
Why?

Reply:
<img width="745" alt="Screen Shot 2021-05-27 at 11 29 31 AM" src="https://user-images.githubusercontent.com/6935300/119870671-d9bd2180-bede-11eb-8584-2a163cdb5988.png">
ah yes because of this, i needed to come back to fix. thanks